### PR TITLE
Sort package imports

### PR DESCRIPTION
**Pull request by lavjain**
Also remove a redundant `make depend` call

### CODE REVIEW

**Comment by shivzone** [comment]
Can you instead run `make format` and commit changes based on that instead of hunting down things manually or using an IDE.

**Comment by shivzone** [comment]
Why do we need the updates to `Gopkg.lock` ?

**Comment by kyeap-pivotal** [comment]
> Why do we need the updates to `Gopkg.lock` ?

Gopkg.lock is usually also updated whenever Gopkg.toml is changed. Gopkg.lock is the output of the dependency solver. Contains information such as the exact commit of the dependency we are using.

File: ci/scripts/test-gpbackup-manager.bash [hunk 1]
```diff
@@ -46,7 +46,6 @@ psql -d postgres -c "create extension pgcrypto"
 
 # Test gpbackup manager
 pushd \$GOPATH/src/github.com/pivotal/gp-backup-manager
-  make depend
```
Comment (kyeap-pivotal) [inline ci/scripts/test-gpbackup-manager.bash]: does this break the pipeline? make depend installs ginkgo. which is required for `make unit`, `integration`, and `end_to_end`

Comment (lavjain) [reply ci/scripts/test-gpbackup-manager.bash]: According to the makefile for gp-backup-manager, `make integration` is already running depend
```
integration : depend
		ginkgo $(GINKGO_FLAGS) integration 2>&1
```

Comment (lavjain) [reply ci/scripts/test-gpbackup-manager.bash]: The dev pipeline is green

**Pull request merged**